Sasquatch/Sasquatch/ViewControllers/Cells/MSAnalyticsPropertyTableViewCell.swift

```diff
@@ -7,4 +7,7 @@ import UIKit
   @IBAction func dismissKeyboard(_ sender: UITextField!) {
     sender.resignFirstResponder()
   }
+    
+    // MARK:variable
+    var curTarget: String = "Child 1"
```

Comment:
What's this one? Is it a default selection?

Reply:
At first, i was expected that the target retrive from code will be a name like "Child 1" or "Child 2", so i set "Child 1" as default, but later found what i got is a Id...

Missed clean up it, will remove from new commit. Thanks~